ci/run_tests.sh

```diff
@@ -21,4 +21,4 @@ pytest $TESTS_DIRS \
     -ra \
     --junitxml=junit.xml \
     --cov=ibis \
-    --cov-report=xml:coverage.xml | tee pytest.log
+    --cov-report=xml:coverage.xml "$@" | tee pytest.log
```

Comment:
Would it make sense to add `-n auto` for all backends?

If we just want to have this for some backends, I'd implement this in the same way as `PYTEST_EXPRESSION`. With an environment variable for the number of processes, or using the `-m ...` in the same way you implemented this with `$@` (just adding the `-m` in the call in the CI). Having two different ways to do the same makes this more difficult to understand.

Reply:
Can you clarify how adding another environment variable is an improvement above using the standard pytest options?

Reply:
I don't have a preference to use an environment variable or `$@`. My point is that what we're doing with `-n` and `-m` is the same, and after this PR, we're using different approaches. Id you prefer to use `$@` as you're doing, I'm fine. But I'd then add `-m ...` to the github yaml, and stop setting the `PYTEST_EXPRESSION` variable. Just for consistency and simplicity.

Reply:
Went ahead and inlined the marker expression.